bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/net/internal/NetworkConfigOptionProvider.java

```diff
@@ -47,6 +49,12 @@
                     .filter(a -> a.getAddress() instanceof Inet4Address);
             return ipv4Addresses.map(a -> new ParameterOption(a.toString(), a.toString())).collect(Collectors.toList());
         }
+
+        if (param.equals(PARAM_BROADCAST_ADDRESS)) {
+            List<String> broadcastAddrList = NetUtil.getAllBroadcastAddresses();
```

Comment:
You are lucky here that `getAllBroadcastAddresses` returns a modifiable list, otherwise your next line would fail. To make sure that your code survives potential refactorings of `NetUtil. getAllBroadcastAddresses()`, better construct your own List here.